Removes Twitter (#454)
springernature/frontend-playbook: writing/social-media.md

> [Cruft.io](http://cruft.io) is a blog run by a group of developers, testers, sysadmins, scrum masters and product owners at Springer Nature. Anyone can post there and the instructions can be found in the [project's wiki (internal)](https://github.com/springernature/cruft/).

Reviewer: ```suggestion
[Cruft.io](http://cruft.io) is a blog run by Springer Nature Technology. Anyone can post there and the instructions can be found in the [project's wiki (internal)](https://github.com/springernature/cruft/).
```
